### PR TITLE
Release 2.0.1

### PR DESCRIPTION
**Pull request by alarreine**

### CODE REVIEW

File: md/release_notes.md [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
 ```
+## What's new in 2.0.1 (2018-08-02)
+
+### Bugfixes
+
+* BCD-274 Fix Docker Engine security configuration.
+
+### Enhancements
 
+* BCD-242 Fix version of all dependencies.
```
Comment (JeremJR) [inline md/release_notes.md]: BCD-242 Fix dependencies version of the license module?

File: md/release_notes.md [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
 * The same BCD stack cannot be managed with multiple BCD controller instances due to the use of Terraform "local" backend.
 * Due to [Ansible issue #35255](https://github.com/ansible/ansible/issues/35255) the warning message "could not match supplied host pattern" is displayed when there is no load balancer required for a noncluster deployment:
 ```
 [WARNING]: Could not match supplied host pattern, ignoring: load_balancer
 ```
+## What's new in 2.0.1 (2018-08-02)
+
+### Bugfixes
+
+* BCD-274 Fix Docker Engine security configuration.
```
Comment (JeremJR) [inline md/release_notes.md]: BCD-274 Security fix : disable Docker daemon tcp socket listening?

File: md/release_notes.md [hunk 3]
```diff
@@ -1,23 +1,22 @@
 # Release notes
 
-## Breaking changes
```
Comment (JeremJR) [inline md/release_notes.md]: The Breaking changes introduced by BCD 2.0 remain true.

**Review by JeremJR — CHANGES_REQUESTED** [review]

File: md/release_notes.md (hunk 3 above)
Comment (alarreine) [reply md/release_notes.md]: [commit](https://github.com/bonitasoft/bonita-continuous-delivery-doc/pull/84/commits/8b3bd90f435ab80f6e4e1e2b0ca0860b17ceff86)

File: md/release_notes.md (hunk 2 above)
Comment (alarreine) [reply md/release_notes.md]: [commit](https://github.com/bonitasoft/bonita-continuous-delivery-doc/pull/84/commits/8b3bd90f435ab80f6e4e1e2b0ca0860b17ceff86)

File: md/release_notes.md (hunk 1 above)
Comment (alarreine) [reply md/release_notes.md]: [commit](https://github.com/bonitasoft/bonita-continuous-delivery-doc/pull/84/commits/8b3bd90f435ab80f6e4e1e2b0ca0860b17ceff86)

**Review by JeremJR — APPROVED** [review]

**Review by JeremJR — APPROVED** [review]

**Pull request merged**